pom.xml

```diff
@@ -330,7 +570,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-gpg-plugin</artifactId>
-            <version>1.6</version>
+            <version>3.2.7</version>
```

Comment:
I hope this won't introduce any surprises when we try perform an actual release to Maven Central. Something to keep in mind for later.

Reply:
Sure, lets try if not we will revert.